test/domains/comet-native/native-children.test.ts

```
@@ -477,6 +478,104 @@ children:
+    });
+    expect(drifted.acceptance_index?.A2?.text).toBe('A stale copy of the second behavior.');
+    expect(nativeChildrenIndexDrift(drifted, acceptanceIds, options)).toEqual({
+      missing: [],
+      extra: [],
+      mismatched: ['A2'],
+      uncovered: [],
+      unknownCovers: [],
+    });
+
+    const staleIds = READABLE_CHILDREN.replace(
+      /  A2:[\s\S]*?children:/u,
```

ESLint

[error] Spaces are hard to count. Use {2}.

(no-regex-spaces)